pkg/scheduler/actions/allocate/allocate.go

```diff
@@ -272,3 +282,38 @@ func (alloc *Action) Execute(ssn *framework.Session) {
 }
 
 func (alloc *Action) UnInitialize() {}
+
+func setPodVolumesByNode(pod *v1.Pod, node *v1.Node, ssn *framework.Session) (*volumescheduling.PodVolumes, error) {
+	client := ssn.KubeClient()
+	informerFactory := informers.NewSharedInformerFactory(client, 0)
+	opts := []runtime.Option{
+		runtime.WithClientSet(client),
+		runtime.WithInformerFactory(informerFactory),
+	}
+	fh, err := runtime.NewFramework(nil, nil, nil, opts...)
+	if err != nil {
+		klog.V(3).Infof("new framework handle instance failed for %v", err.Error())
+		return nil, err
+	}
+	pl, err := volumebinding.New(&config.VolumeBindingArgs{
+		BindTimeoutSeconds: 300,
```

Comment:
if binding timeout happen, it will block other task for 300s. I think 60s is enough.